### PR TITLE
MongoDB monitoring showcase

### PR DESCRIPTION
**Pull request by weixiang1862**

### CODE REVIEW

**Comment by weixiang1862** [comment]
UI image is not the latest, it shows i18n key now.

image

**Comment by wu-sheng** [comment]
Do you mean UI having issues?

**Comment by weixiang1862** [comment]
> Do you mean UI having issues?

No issue，I think

> Do you mean UI having issues?

No issue, it work on my local with command `npm run dev`.
May be the image with this tag does not contains the latest code, but it is latest tag I found in registry.

File: Makefile.in
Comment (wu-sheng) [inline Makefile.in:32]: https://github.com/apache/skywalking/actions/runs/5621499911

You didn't check, this commit doesn't push the image successfully, I just rerun it.
Let's see.

Comment (wu-sheng) [reply Makefile.in:32]: Please retry, the images are ready now.

**Comment by weixiang1862** [comment]
Main repo need sync ui from booster-ui.

**Comment by wu-sheng** [comment]
You could sync through a PR. Update UI sub module.

**Comment by weixiang1862** [comment]
> You could sync through a PR. Update UI sub module.

ok.

**Comment by weixiang1862** [comment]
Could you please build a new ui image? I don't know how to trigger it.

**Comment by wu-sheng** [comment]
It is automatically built after merged

Now it is there, https://github.com/apache/skywalking/pkgs/container/skywalking%2Fui

**Comment by weixiang1862** [comment]
> It is automatically built after merged
> 
> Now it is there, https://github.com/apache/skywalking/pkgs/container/skywalking%2Fui

Ok, thanks. I will try this.

**Comment by weixiang1862** [comment]
It works now, thanks.

**Comment by wu-sheng** [comment]
Wait for @kezhenxu94 and @wankai123 review.

**Review by kezhenxu94 — APPROVED** [review]
How do the metrics charts look like? Do we need to add some traffic reads/writes to the MongoDB?

**Comment by weixiang1862** [comment]
> How do the metrics charts look like? Do we need to add some traffic reads/writes to the MongoDB?
The exporter it self can provider some traffic，I will pin a picture when I back.

**Comment by weixiang1862** [comment]
@kezhenxu94 It looks like this, I running it on my local minikube for about 10 hours.

image

image

**Review by kezhenxu94 — APPROVED** [review]

File: deploy/platform/kubernetes/templates/feature-mongodb/opentelemetry-config.yaml
Comment (weixiang1862) [reply deploy/platform/kubernetes/templates/feature-mongodb/opentelemetry-config.yaml:29]: It was hardcoded in chart template, and can't   be replaced by values.yaml, are there other ways to change this name?

Comment (wankai123) [reply deploy/platform/kubernetes/templates/feature-mongodb/opentelemetry-config.yaml:29]: @kezhenxu94 Is that possible to override the name?

Comment (weixiang1862) [reply deploy/platform/kubernetes/templates/feature-mongodb/opentelemetry-config.yaml:29]: I can try to add a `__meta_kubernetes_pod_controller_name` label, then this scrape job can be limited under mongodb case.

**Review by wankai123 — APPROVED** [review]

**Comment by wu-sheng** [comment]
@wankai123 When you have time, let's bring this to demo env as well.

Doc is hosted, https://skywalking.apache.org/docs/main/next/en/setup/backend/backend-mongodb-monitoring/

**Pull request merged**
